ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -65,6 +69,29 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-gcp
+- as: operator-e2e-latest-osus
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: e2e-test
+      cli: initial
+      commands: |
+        make func-test \
+        OPENSHIFT_BUILD_NAMESPACE= \
+        RELATED_IMAGE_OPERAND=registry.ci.openshift.org/cincinnati-ci/cincinnati-build-root:deploy
```

Comment:
`latest-osus` isn't particularly clear to me.  Looks like you're floating the Cincinnati image here, and then also floating the `graph-data` image below?  Instead of hard-coding `registry.ci.openshift.org/...`, can we use a `base_images` stanza to have the CI tools pull in the Cincinnati you want?

Reply:
>latest-osus isn't particularly clear to me

perhaps renaming to `operator-e2e-stable-operand` and `operator-e2e-latest-operand` is less confusing?

>  and then also floating the graph-data image below?

we're always floating graph-data (it is still being built on the fly for the test, but I'll remove it in a follow up PR)

> can we use a base_images stanza to have the CI tools pull in the Cincinnati you want?

I don't think so - we need an image pullspec and ci-operator can resolve a tag for us - but I don't think it can replace an env var - I'll give it a try though